rollup/fees/rollup_fee.go

```diff
@@ -206,9 +214,9 @@ func VerifyFee(signer types.Signer, tx *types.Transaction, state StateDB) error
 		return errors.New("invalid transaction: insufficient funds for gas * price + value")
 	}
 
-	cost = cost.Add(cost, l1Fee)
+	cost = cost.Add(cost, l1DataFee)
 	if balance.Cmp(cost) < 0 {
-		return errors.New("invalid transaction: insufficient funds for l1fee + gas * price + value")
+		return errors.New("invalid transaction: insufficient funds for l1DataFee + gas * price + value")
```

Comment:
This is also a breaking change and some projects like ChainLink might already rely on this error message. We could change it any notify them but I think it's easier to keep it as `l1fee`.

Reply:
I will revert this one

Reply:
fixed in https://github.com/scroll-tech/go-ethereum/pull/351/commits/2753504c3518df5f66ecad89caf7e967d02dbb4c